src/app/AppContainer.js

```diff
@@ -54,6 +56,12 @@ class AppContainer extends React.Component<PropsType> {
 		}
 	}
 
+	componentDidMount = () => {
+		const blocks = this.props.getBlocks();
+		const hasUnsupportedBlocks = ! isEmpty( blocks.filter( ( { name } ) => name === unsupportedBlockName ) );
+		RNReactNativeGutenbergBridge.editorDidMount( hasUnsupportedBlocks );
```

Comment:
Oh, this direct call will not work on the demo app for Android, since there is no support for the native side of the bridge in the Android demo app.

It's also the reason the tests fail for Android.

I will open a separate PR against this one to add support in the demo app.

Reply:
Done, opened #589.

Regarding this failing tests in this PR, let's add a mock for `editorDidMount` in `jest/setup.js`. Let me know if you want me to add the mock.

Reply:
Mock added in ded52a48545d007da384ee05f2b545a17c0dfcc6.